components/mixins/experience-event/experienceevent-marketing.schema.json

```diff
@@ -20,6 +23,21 @@
           "title": "Marketing",
           "$ref": "https://ns.adobe.com/xdm/context/marketing",
           "description": "The information related to marketing activities that are active with the touchpoint."
+        },
+        "xdm:internalMarketing": {
+          "title": "Internal Marketing",
+          "$ref": "https://ns.adobe.com/xdm/context/marketing",
+          "description": "Internal Marketing data type that contains group, name, and code."
+        },
+        "xdm:productFindingMethod": {
+          "title": "Product Finding Method",
+          "type": "string",
+          "description": "How the user found a product immediately prior to viewing it. Allows a view of which finding methods were most effective for each product."
+        },
+        "xdm:trackingClick": {
```

Comment:
If this property is supposed to be at the root of the Marketing mixin, can this property be part of the marketing data type ?

Reply:
marketing datatype is currently limited to three common fields namely "group, name, and code".  Not very sure where else is marketing datatype used to analyze the impact if we add couple of more fields to the datatype itself.

Reply:
@mjasrotia @prabhum2
Let's move this to a V2 change and just approve the meta:tags for now?